fedimint-logging/src/lib.rs

```diff
@@ -19,6 +19,7 @@ pub const LOG_TASK: &str = "task";
 pub const LOG_WALLET: &str = "wallet";
 pub const LOG_TEST: &str = "test";
 pub const LOG_ECASH_RECOVERY: &str = "ecash-recovery";
+pub const LOG_BITCOIND: &str = "bitcoind";
```

Comment:
Not sure if this is waht we want to go with it. Generally we want to have "categories" of stuff here. `net::blockchain` or something maybe?

Reply:
Should this be refactored into an enum?

Reply:
Would a nested enum work?
```rs
enum LogTarget {
    Net(NetTarget),
    ....
}

enum NetTarget {
    Blockchain,
    ....
}
```

Reply:
Logging targets must be strings, AFAIK, and generally correspond (and default to) Rust module naming, so `foo:bar::xyz` etc. They are only used for logging filtering via `RUST_LOG`. No point in trying to make them anything other than string contants, AFAICT.

Reply:
With enums it looks like we'd have to impl Display for each variation to convert to strings. Probably not worth it when we can just keep this section of constants tidy and organized.

Reply:
Coming back to the beginning of this discussion, I think a rename to `net::blockchain` would make sense.

Reply:
😅
oh yea...
done.